scripts/mirror.js

```diff
@@ -219,71 +219,61 @@ const bumpChromeAndroid = (originalData, sourceData, source) => {
 };
 
 /**
- * @param {SupportStatement} originalData
- * @param {SupportStatement} sourceData
- * @param {string} source
+ * @param {SupportStatement} comp
  * @returns {SupportStatement}
  */
-const bumpEdge = (originalData, sourceData, source) => {
-  let newData = {};
-
-  if (source == 'ie') {
-    if (sourceData.version_removed && sourceData.version_removed !== null) {
-      newData.version_added = false;
-    } else if (sourceData.version_added !== null) {
-      newData.version_added = sourceData.version_added ? '12' : null;
-    }
+const bumpEdge = comp => {
+  let newData = comp['edge'];
+  let originalData = comp['edge'];
+  let ieData = comp['ie'];
+  let chromeData = comp['chrome'];
 
-    if (sourceData.notes) {
-      newData.notes = updateNotes(
-        sourceData.notes,
-        /Internet Explorer/g,
-        'Edge',
-      );
-    }
-  } else if (source == 'chrome') {
-    newData = originalData == undefined ? sourceData : originalData;
+  if (ieData.version_removed && ieData.version_removed !== null) {
+    newData.version_added = false;
+  } else if (ieData.version_added !== null) {
+    newData.version_added = ieData.version_added ? '12' : null;
+  }
 
-    let chromeFalse =
-      sourceData.version_added === false ||
-      sourceData.version_removed !== undefined;
-    let chromeNull = sourceData.version_added === null;
+  let chromeFalse =
+    chromeData.version_added === false ||
+    chromeData.version_removed !== undefined;
+  let chromeNull = chromeData.version_added === null;
 
-    if (originalData === undefined) {
-      newData.version_added = chromeFalse ? false : chromeNull ? null : '≤79';
-    } else {
-      if (!chromeFalse && !chromeNull) {
-        if (originalData.version_added == true) {
-          newData.version_added = '≤18';
-        } else {
-          if (
-            sourceData.version_added == true ||
-            Number(sourceData.version_added) <= 79
-          ) {
-            if (originalData.version_added == false) {
-              newData.version_added = '79';
-            } else if (originalData.version_added == null) {
-              newData.version_added = '≤79';
-            }
-          } else {
-            newData.version_added == sourceData.version_added;
+  if (originalData === undefined) {
+    newData.version_added = chromeFalse ? false : chromeNull ? null : '≤79';
+  } else {
+    if (!chromeFalse && !chromeNull) {
+      if (originalData.version_added == true) {
+        newData.version_added = '≤18';
+      } else {
+        if (
+          chromeData.version_added == true ||
```

Comment:
Increasingly, I think version_added true for Chrome means we don't know enough. What we *could* infer if Chrome is true and IE is false is really "≥79", but that's not an option. In the absence of that, I think we just have to set Edge to true as well?